java/src/main/java/com/powsybl/dataframe/network/NetworkDataframes.java

```diff
@@ -268,6 +269,18 @@ static NetworkDataframeMapper shuntsNonLinear() {
                 .build();
     }
 
+    static NetworkDataframeMapper linearShuntsSections() {
+        Function<Network, Stream<Pair<String, ShuntCompensatorLinearModel>>> linearShunts = network ->
+                network.getShuntCompensatorStream()
+                        .filter(sc -> sc.getModelType() == ShuntCompensatorModelType.LINEAR)
+                        .map(shuntCompensator -> Pair.of(shuntCompensator.getId(), (ShuntCompensatorLinearModel) shuntCompensator.getModel()));
+        return NetworkDataframeMapperBuilder.ofStream(linearShunts)
+                .stringsIndex("id", Pair::getLeft)
+                .doubles("g_per_section", p -> p.getRight().getGPerSection())
```

Comment:
We should add setters too!

It would be nice to be able to set the maximum section count too. I suggest to also add a getter for it in that table, so that it is consistent.